feat: enhance cryptkeeperzk/provider (#578)
CryptKeeperZK/crypt-keeper-extension: packages/providers/README.md

# `@cryptkeeperzk/providers`

> The `@cryptkeeperzk/providers` package provides **CryptKeeper** injected providers that enable applications to easily connect and communicate with the CryptKeeper browser extension.

Reviewer: ```suggestion
The `@cryptkeeperzk/providers` package provides **CryptKeeper** injected providers which allow applications to easily connect and communicate with the CryptKeeper browser extension.
```
